icacls_work.rb

```diff
@@ -1,36 +1,38 @@
-class Rights
+class Permission
 
   attr_reader :path, :name
   def initialize(path, name)
     @path = path
     @name = name
   end
 
-  def get_res_string(name_file)
-    system("cd #{@path} && icacls #{@name} > #{name_file}.txt")
-    written_file = File.open("#{@path}\\#{name_file}.txt")
-    written_file.each do |line|
-      return line if line.include?'Users'
+  def users_string
```

Comment:
It's not clear for me, what this method actually do.. :)

Reply:
Where did you find it? I've deleted this code 
I don't use any files now

Reply:
Red lines already removed :)
I'm talking about users_string method

Reply:
Okay, I try to explain: this method return the string that contains smth like this Users:(I)(RX)
In this string you can see all permissions that have Users. This string I will use in order to find (RX) access and when display it

Reply:
I mean, method should describe some action like get_string, read_something_from_string. Ideally, the best name for the method is when a person who does not know the code is able to understand what the method is doing.

Reply:
Got it

Reply:
done